src/utils/get-functions.js

```diff
@@ -24,4 +26,29 @@ const getFunctions = async (functionsSrcDir) => {
   return functionsWithProps
 }
 
-module.exports = { getFunctions }
+const getWatchDirs = (functionsSrcDir, functions) => {
+  const localFilesDirs = functions
```

Comment:
This can cause duplicate events if someone imports from both `<repo>/utils` and `<repo>/utils/common`.

Not sure how to handle it

Reply:
Is this still the case, or did the debounced events fix this?

Reply:
We'll still get duplicate events, but we'll handle them once